envoyauth/request_test.go

```diff
@@ -73,7 +73,7 @@ func TestGetParsedBody(t *testing.T) {
 			  "headers": {
 				"content-type": "application/json"
 			  },
-			  "body": "foo"
+			  "body": "\"foo\""
```

Comment:
💭 this is interesting... I think it's correct _this way_, but would this mean that the behaviour changes in some what that's unrelated to escaped characters...?

Reply:
Yes, that's true, but if foo is directly in the body, I don't think it's a json literal (it looks like `true` or `false`).
The reason why it was working so far is that util.Unmarshal assumed it was yaml and converted `foo` to `"foo"`. (It seems that yaml considers `foo` as a string even without quoting).
Thus, I thought that `foo` would not pass the test, but should be converted to `"foo"`.

Reply:
Ah, gotcha. So, having a plain "foo" body (not "\"foo\"") actually is a case of sending YAML (not JSON).  And that's unlikely to be sent by anyone using an application/json header anyways.

I think we're OK here, we should, however, point out that we had erroneously been accepting YAML with a JSON header in the next _Release Notes_ 🎗️

Reply:
You're right.
The release notes seem to use the github feature.
Is there anything I can do to help?

Reply:
It's cool, thanks, I'll take care of making a note for it. 😃